.changes/1885-keyboard-fixes.md

```diff
@@ -0,0 +1 @@
+- Minor fix to dismiss the keyboard after submitting a comment & making the icons on the image overlay accessible regardless of top bezels
```

Comment:
Just to be sure, this also fixes the bug on pin editing area? this is more likely an iOS keyboard bug...

Reply:
no, I am not aware of any pin editing area bug, as I wasn't able to reproduce it...